docs/terms/expectation.md

```diff
@@ -3,10 +3,134 @@ title: Expectation
+
+## Use cases
+
+<CreateHeader/>
+
+Expectations are obviously a fundamental component of the Create Expectations step in working with Great Expectations.  It is when you are creating Expectations that you will have the most direct interaction with them.  For further information on this process, please see [our overview on the Create Expectations process](../guides/expectations/create_expectations_overview.md), and [our related how-to guides](../guides/expectations/index.md).
+
+
+<ValidateHeader/>
+
+When you create your Checkpoints, you will be able to configure them to use specific Expectation Suites.  Other than setting up this configuration (or arranging to pass Expectation Suites at runtime) you will not need to directly interact with the Expectations themselves.  Instead, when you run your Checkpoint it will handle using the Expectations in any of its Expectation Suites to Validate the data indicated in its Batch Request/s.  This will be done under the hood, with the <TechnicalTag relative="../" tag="validation_result" text="Validation Results" /> that are generated being passed along to the Checkpoint's (optional) <TechnicalTag relative="../" tag="action" text="Actions" /> for further processing.
+
+
+## Features
+
+### Versatility out of the box
+
+Great Expectations' built-in library includes more than 50 common Expectations, such as:
+
+* `expect_column_values_to_not_be_null`
+* `expect_column_values_to_match_regex`
+* `expect_column_values_to_be_unique`
+* `expect_column_values_to_match_strftime_format`
+* `expect_table_row_count_to_be_between`
+* `expect_column_median_to_be_between`
+
+For a full list of available Expectations, please check out the [Expectation Gallery](https://greatexpectations.io/expectations).
+
+### Customization
+
+Expectations are especially useful when they capture critical aspects of data understanding that analysts and practitioners know based on its *semantic* meaning. It's common to want to extend Great Expectations with application or domain specific Expectations. For example:
+
+```bash
+expect_column_text_to_be_in_english
+expect_column_value_to_be_valid_icd_code
+```
+
+These Expectations aren't included in the default set, but could be very useful for specific applications.
+
+Fear not! Great Expectations is designed for customization and extensibility.
+
+Building custom Expectations is easy and allows your custom logic to become part of the validation, documentation, and even profiling workflows that make Great Expectations stand out. See the guide on [creating custom Expectations](/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_expectations) for more information on building Expectations and updating <TechnicalTag relative="../" tag="data_context" text="Data Context" /> configurations to automatically load <TechnicalTag relative="../" tag="batch" text="Batches" /> of data with custom <TechnicalTag relative="../" tag="data_asset" text="Data Assets" />Data Assets.
```

Comment:
Just flagging that the links to creating Expectations docs may need to change in light of the work that is being done there.
```suggestion
Building custom Expectations is easy and allows your custom logic to become part of the validation, documentation, and even profiling workflows that make Great Expectations stand out. See the guide on [creating custom Expectations](/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_expectations) for more information on building Expectations and updating <TechnicalTag relative="../" tag="data_context" text="Data Context" /> configurations to automatically load <TechnicalTag relative="../" tag="batch" text="Batches" /> of data with custom <TechnicalTag relative="../" tag="data_asset" text="Data Assets" />.
```